include/line_numbers.h

```diff
@@ -0,0 +1,52 @@
+#ifndef LINE_NUMBERS_H
+#define LINE_NUMBERS_H
+
+// The macros in this file serve to reduce clutter from version differences that are only due to line numbers being different.
+
+/**
+ * Macro that defines line numbers by version.
+ * This macro is not meant to be used externally, see wrappers below.
+ */
+#define LN(ntsc_1_0, ntsc_1_1, ntsc_1_2, pal_1_0, pal_1_1, gc_jp, gc_jp_mq, gc_us, gc_us_mq, gc_eu, gc_eu_mq, gc_jp_ce, debug) \
```

Comment:
I think this should be the same order as versions.h so the version ordering is consistent everywhere